src/catkin_pkg/cli/prepare_release.py

```diff
@@ -194,10 +195,16 @@ def push_changes(base_path, vcs_type, tag_name, dry_run=False):
 
 
 def _find_executable(vcs_type):
-    for path in os.getenv('PATH').split(os.path.pathsep):
-        file_path = os.path.join(path, vcs_type)
-        if os.path.isfile(file_path):
+    try:
+        file_path = shutil.which(vcs_type)
+        if file_path is not None:
             return file_path
```

Comment:
Please move the last two lines out of the `try` / `except` block since they can't raise an `AttributeError`

Reply:
I switched this over to a better approach. It's more obvious to catch an ImportError